src/main/java/controllers/CreateUserandFridgeController.java

```diff
@@ -0,0 +1,24 @@
+package controllers;
+
+import useCases.CreateUserAndFridge.CreateUserAndFridgeInputBoundry;
+import useCases.CreateUserAndFridge.CreateUserandFridgeRequestModel;
+import useCases.CreateUserAndFridge.CreateUserandFridgeResponseModel;
+
+public class CreateUserandFridgeController{
```

Comment:
Please add some doc string comments to this class.

Reply:
Added Extra docstrings